Index: src/main/java/com/science/gtnl/Utils/LateMixins.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.science.gtnl.Utils;\r\n\r\nimport com.gtnewhorizon.gtnhmixins.ILateMixinLoader;\r\nimport com.gtnewhorizon.gtnhmixins.LateMixin;\r\n\r\nimport java.util.Arrays;\r\nimport java.util.List;\r\nimport java.util.Set;\r\n\r\n@LateMixin\r\npublic class LateMixins implements ILateMixinLoader {\r\n\r\n    @Override\r\n    public String getMixinConfig() {\r\n        return \"mixins.EyeOfHarmonyBuffer.late.json\";\r\n    }\r\n\r\n    @Override\r\n    public List<String> getMixins(Set<String> loadedMods) {\r\n        return Arrays.asList(\r\n            \"MTETieredMachineBlockMixin\",\r\n            \"Accessor.MTEMegaBlastFurnaceAccessor\"\r\n        );\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/main/java/com/science/gtnl/Utils/LateMixins.java b/src/main/java/com/science/gtnl/Utils/LateMixins.java
--- a/src/main/java/com/science/gtnl/Utils/LateMixins.java	(revision f841d622f55ad15b6fba40d1d7bdc2068becf9a2)
+++ b/src/main/java/com/science/gtnl/Utils/LateMixins.java	(date 1733640780015)
@@ -12,7 +12,7 @@
 
     @Override
     public String getMixinConfig() {
-        return "mixins.EyeOfHarmonyBuffer.late.json";
+        return "mixins.GTNotLeisurel.late.json";
     }
 
     @Override
diff --git a/src/main/resources/mixin/mixins.gtnl.late.json b/src/main/resources/mixins.GTNotLeisurel.late.json
rename from src/main/resources/mixin/mixins.gtnl.late.json
rename to src/main/resources/mixins.GTNotLeisurel.late.json
diff --git a/src/main/resources/mixin/mixins.gtnl.json b/src/main/resources/mixins.GTNotLeisure.json
rename from src/main/resources/mixin/mixins.gtnl.json
rename to src/main/resources/mixins.GTNotLeisure.json
